### PR TITLE
Update Jenkins minor version to 2.46.x

### PR DESCRIPTION
**Pull request by jtescher**

### CODE REVIEW

**Comment by bparees** [comment]
[test]

File: 2/Dockerfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ EXPOSE 8080 50000
 RUN curl https://pkg.jenkins.io/redhat-stable/jenkins.repo -o /etc/yum.repos.d/jenkins.repo && \
     rpm --import https://pkg.jenkins.io/redhat-stable/jenkins-ci.org.key && \
     yum install -y centos-release-scl-rh && \
-    INSTALL_PKGS="rsync gettext git tar zip unzip java-1.8.0-openjdk java-1.8.0-openjdk.i686 java-1.8.0-openjdk-devel java-1.8.0-openjdk-devel.i686 jenkins-2.32.3-1.1" && \
+    INSTALL_PKGS="rsync gettext git tar zip unzip java-1.8.0-openjdk java-1.8.0-openjdk.i686 java-1.8.0-openjdk-devel java-1.8.0-openjdk-devel.i686 jenkins-2.46.1-1.1" && \
```
Comment (bparees) [inline 2/Dockerfile]: please update the env on line 17 too, and the display name on line 23.

Comment (jtescher) [inline 2/Dockerfile]: @bparees done ✅

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
thanks @jtescher !

**Pull request merged**